server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/http/SessionRoutes.scala

```diff
@@ -58,12 +59,18 @@ class SessionRoutes @Inject() (@Named(InjectionKeys.RFC_8621) val authenticator:
       .subscribeOn(Schedulers.elastic())
       .asJava()
 
+  private val redirectToSession: JMAPRoute.Action = JMAPRoutes.redirectTo(JMAP_SESSION)
+
   override def routes: Stream[JMAPRoute] =
     Stream.of(
       JMAPRoute.builder()
         .endpoint(new Endpoint(HttpMethod.GET, JMAP_SESSION))
         .action(generateSession)
         .corsHeaders,
+      JMAPRoute.builder()
+        .endpoint(new Endpoint(HttpMethod.GET, WELL_KNOWN_JMAP))
+        .action(redirectToSession)
+        .corsHeaders,
       JMAPRoute.builder()
         .endpoint(new Endpoint(HttpMethod.OPTIONS, AUTHENTICATION))
```

Comment:
@chibenwa is `AUTHENTICATION` here an error? this should be `JMAP_SESSION` instead?

Reply:
Of course it is ....

Reply:
Here is a fix ;-)

https://github.com/linagora/james-project/pull/3918